src/dali_executor/dali_executor.cc

```diff
@@ -55,29 +55,59 @@ IDescr DaliExecutor::ScheduleInputCopy(const IDescr& input) {
   assert(input.buffers.size() > 0);
   IOBufferI* buffer;
   if (input.buffers[0].device == device_type_t::CPU) {
-    buffer = &cpu_buffers_[input.meta.name];
+    buffer = &cpu_buffers_[input.meta.name + "_inp"];
```

Comment:
It might be better to have a GetCpuInputBuffer or something rather than use a magic suffix string.

Reply:
It can even be `IOBufferI* GetInputBuffer(IDescr &, device_type_t);`

Reply:
Added GetInputBuffer and GetOutputBuffer